README.md

```diff
@@ -56,6 +63,50 @@ This option allows you to disable it if you want:
 
 ```cordova run android ios -- --livereload --ghostMode=false```
```

Comment:
Separating the two is true from the TACO side (https://github.com/Microsoft/TACO.git).
Due to an issue with cordova not passing through the ```--livereload``` flag when emulating (```cordova emulate android -- --livereload```), we had to rely on looking at the process.args directly instead of relying on the plugin context. Because of that, inside the plugin, we make --livereload be synonymous with --devicesync. but from the TACO point of view, they are different.